### ISSUE TITLE
Rename `$ask_follow_up` to `$ask_followup` or `$ask_followup_to`

### ISSUE THREAD

**Issue by whitney-cumber** [neomutt/neomutt#3602]
The config variable `ask_follow_up` should be named `ask_followup` (or even better `ask_followup_to`) to be in line with

a) the actual header name which is "Followup-To"  (and not "Follow-Up-To")
b) be consistent with `$followup_to_poster`

## Expected Behaviour

Consistent naming scheme.

## Actual Behaviour

Inconsistent naming scheme

**Comment by flatcap** [comment]
Yes, `$ask_followup_to` sounds reasonable.
The name came from the original NNTP patch.

Would you like to make the change?
(Please would you make the change :-))

A quick `grep` shows just four files that need altering:
```
docs/config.c
docs/manual.xml.head
send/config.c
send/send.c
```

Plus we'd need a synonym creating (for backwards compatibility) in `send/config.c`, linking the old name to the new, e.g.
```c
{ "askbcc", DT_SYNONYM, IP "ask_bcc", IP "2021-03-21" },
```

Thanks! :heart:

**Comment by whitney-cumber** [comment]
Will come in a couple of minutes.

**Comment by whitney-cumber** [comment]
Fix in PR #3604.
